samples/bot-proactive-messaging-teamsfx/bot/teamsBot.js

```diff
@@ -17,9 +22,10 @@ class TeamsBot extends TeamsActivityHandler {
 
     this.onMembersAdded(async (context, next) => {
       const membersAdded = context.activity.membersAdded;
+      
       for (let cnt = 0; cnt < membersAdded.length; cnt++) {
         if (membersAdded[cnt].id !== context.activity.recipient.id) {
-          const welcomeMessage = 'Welcome to the Proactive Bot sample.  Navigate to http://{your-domain}/api/notify to proactively message everyone who has previously messaged this bot.';
+          const welcomeMessage = `Welcome to the Proactive Bot sample.  Navigate to ${process.env.PROVISIONOUTPUT__BOTOUTPUT__SITEENDPOINT}/api/notify to proactively message everyone who has previously messaged this bot.`;
```

Comment:
Will it work without provisioning?
Support I am not running the app using toolkit and trying it locally without provisioning (Normal app running using Ngrok/dev tunnel)?

Reply:
Please test it locally without toolkit once.

Reply:
This sample is itself made using toolkit and will not work with local setup